api/src/modules/logging/logging.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Logger, Module } from '@nestjs/common';
+
+@Module({
```

Comment:
Probably worth making it global since it's a domain agnostic module, we should not care about code verbosity for this context